packages/lesswrong/lib/collections/posts/views.ts

```diff
@@ -445,18 +445,6 @@ ensureIndex(Posts,
     name: "posts.afSticky_score",
   }
 );
-ensureIndex(Posts,
-  augmentForDefaultView({ metaSticky:-1, score:-1 }),
-  {
-    name: "posts.metaSticky_score",
-  }
-);
-ensureIndex(Posts,
-  augmentForDefaultView({ userId: 1, hideAuthor: 1, ...stickiesIndexPrefix, score:-1 }),
-  {
-    name: "posts.userId_stickies_score",
```

Comment:
This index hasn't been used in the past week. It was re-applied 4 years ago by Jim in a PR entitled "Restore view that was misidentified as unused" (as one of several views that got restored).